apps/web/src/server/applications/pages/case/documentation/applications-documentation.service.js

```diff
@@ -50,19 +52,30 @@ export const getCaseDocumentationFolderPath = (caseId, folderId) => {
  *
  * @param {number} caseId
  * @param {number} folderId
- * @returns {Promise<DocumentationFile[]>}
+ * @param {number} pageSize
+ * @param {number} pageNumber
+ * @returns {Promise<PaginatedDocumentationFiles>}
  */
-export const getCaseDocumentationFilesInFolder = (caseId, folderId) => {
+export const getCaseDocumentationFilesInFolder = (
+	caseId,
+	folderId,
+	pageSize = 50,
+	pageNumber = 0
+) => {
 	// TODO: Mock Version - to be replaced when API to get documents exists:
 	/** @type {DocumentationFile[] } */
-	const documentationFiles = [];
+	const documentationFiles = fixtureDocumentationFiles;
```

Comment:
worth updating the tests to check for correct number of files

Reply:
this is the mocked service (should be replaced by the api). the test for the correct number is in the controller